tests/positive/security/tls.go

```diff
@@ -89,9 +163,22 @@ AKbyaAqbChEy9CvDgyv6qxTYU+eeBImLKS3PH2uW5etc/69V/sDojqpH3hEffsOt
 			}
 		}`)
 
+	customCAServerFile1 = []byte(`{
```

Comment:
Is an entirely new file required for the CA bundle test?  Could the `customCAServerFile` just be reused?

Reply:
NM...I see the need.